services/galley/src/Galley/API/Push.hs

```diff
@@ -80,25 +78,25 @@ newMessagePush botMap mconn mm userOrBots event =
 runMessagePush ::
   forall x r.
   ( Member ExternalAccess r,
-    Member GundeckAccess r,
-    Member TinyLog r
+    Member TinyLog r,
+    Member NotificationSubsystem r
   ) =>
   Local x ->
   Maybe (Qualified ConvId) ->
   MessagePush ->
   Sem r ()
 runMessagePush loc mqcnv mp@(MessagePush _ _ _ botMembers event) = do
-  push (toPush mp)
+  push $ maybeToList $ toPush mp
   for_ mqcnv $ \qcnv ->
     if tDomain loc /= qDomain qcnv
       then unless (null botMembers) $ do
         warn $ Log.msg ("Ignoring messages for local bots in a remote conversation" :: ByteString) . Log.field "conversation" (show qcnv)
       else deliverAndDeleteAsync (qUnqualified qcnv) (map (,event) botMembers)
 
-toPush :: MessagePush -> Maybe Push
+toPush :: MessagePush -> Maybe PushToUser
```

Comment:
Does this need a new name given the return type change?

Reply:
The `PushToUser` type has a bad name. I want to try and take out polymorphism from the `PushTo` type, I think its not needed. Once that happens `PushToUser` can just be `Push`. I want to unify all of these `Push` types. I think they create more confusion than anything.